specification/securitycenter/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/InformationProtectionPolicies/CreateOrUpdateInformationProtectionPolicy_example.json

```diff
@@ -0,0 +1,238 @@
+{
+    "parameters": {
+        "api-version": "2017-08-01-preview",
+        "scope": "providers/Microsoft.Management/managementGroups/148059f7-faf3-49a6-ba35-85122112291e",
+        "informationProtectionPolicyName": "default",
+        "InformationProtectionPolicy": {
+            "id": "/providers/Microsoft.Management/managementGroups/148059f7-faf3-49a6-ba35-85122112291e/providers/Microsoft.Security/informationProtectionPolicies",
+            "name": "default",
+            "type": "Microsoft.Security/informationProtectionPolicies",
+            "properties": {
+                "sensitivityLabels": [
+                    {
+                        "id": "1345da73-bc5a-4a8f-b7dd-3820eb713da8",
+                        "name": "Public",
+                        "order": 100
+                    },
+                    {
+                        "id": "7aa516c7-5a53-4857-bc6e-6808c6acd542",
+                        "name": "General",
+                        "order": 200
+                    },
+                    {
+                        "id": "575739d2-3d53-4df0-9042-4c7772d5c7b1",
+                        "name": "Confidential",
+                        "order": 300
+                    },
+                    {
+                        "id": "167a46e8-dfb3-4fb8-920c-1a4c8bf69b75",
+                        "name": "Confidential - GDPR",
+                        "order": 400
+                   },
+                    {
+                        "id": "bc8aaa92-d351-4931-9e7f-ad3acf58680c",
+                        "name": "Highly Confidential",
+                        "order": 500
+                    },
+                    {
+                        "id": "1bb5d576-8849-4995-80aa-97520f92e234",
+                        "name": "Highly Confidential - GDPR",
+                        "order": 600
+                    }
+                ],
+                "sensitivityInformationTypes": [
+                    {
+                        "id": "7fb9419d-2473-4ad8-8e11-b25cc8cf6a07",
+                        "name": "Networking",
+                        "order": 100,
+                        "recommendedLabelId": "575739d2-3d53-4df0-9042-4c7772d5c7b1"
```

Comment:
How would `recommendedLabelId` be generated in an ARM template? Wouldn't this become a non-idempotent PUT in an ARM template?

Reply:
This is not generated, it should reference the corresponding recommended label.
User will set and ID of a specific label, then he'll reuse that id here